examples/gno.land/p/demo/audio/wav/wav.gno

```diff
@@ -0,0 +1,100 @@
+package wav
```

Comment:
I know I'm more annoying than golint, but I would also appreciate the same here :)

Reply:
No worries! I sometimes work so fast I forget about these things, added in 97081f64